libwebauthn/src/proto/ctap2/model.rs

```diff
@@ -644,26 +644,32 @@ impl Ctap2GetInfoResponse {
             (self.option_enabled("pinUvAuthToken") && self.option_enabled("uv"))
     }
 
-    pub fn uv_operation(&self) -> Ctap2UserVerificationOperation {
-        if self.option_enabled("uv") {
+    pub fn uv_operation(&self, uv_blocked: bool) -> Option<Ctap2UserVerificationOperation> {
```

Comment:
This method is getting a bit complex - may be worth refactoring this soon.

Reply:
Let's make an issue for this

Reply:
Created #60